frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -36,59 +36,76 @@ const NotificationIcon: React.FC<NotificationIconTypes> = ({ type }) => {
   }
 };
 
+const NotificationAction: React.FC<NotificationActionProps> = ({ drawerToggle, text, url }) => (
+  <div className="pf-c-notification-drawer__header-action">
+    <Link
+      to={url}
+      onClick={(e) => {
+        e.stopPropagation();
```

Comment:
Do we need `stopPropagation`? Can this just be `onClick={drawerToggle}`?

Reply:
@spadgett , yeah the stopPropagation is needed as far as i can see. i can't get both the drawer to close and to follow the action path rather than the target path without this.